samples/charts/data-chart/composite-chart/src/index.tsx

```diff
@@ -15,103 +28,201 @@ IgrCalloutLayerModule.register();
+              overlap="0"
+              gap="0.1"
+              useClusteringMode={true}
+            />
+
+            <IgrNumericYAxis
+              name="yAxis"
+              abbreviateLargeNumbers={true}
+              title="U.S. Dollars"
+              minimumValue={0}
+              maximumValue={30000000}
+              titleLeftMargin={5}
+              titleRightMargin={0}
+            />
+            <IgrNumericYAxis
+              name="yAxis2"
+              abbreviateLargeNumbers={true}
+              title="Percentage"
+              majorStrokeThickness={0}
+              minimumValue={20}
+              maximumValue={80}
+              labelLocation="OutsideRight"
+              labelHorizontalAlignment="Left"
+            />
+
+            <IgrColumnSeries
+              name="series1"
+              xAxisName="xAxis"
+              yAxisName="yAxis"
+              valueMemberPath="Revenue"
+              title="Revenue"
+              markerType="Hidden"
+            />
+
+            <IgrColumnSeries
+              name="series2"
+              xAxisName="xAxis"
+              yAxisName="yAxis"
+              valueMemberPath="Expenses"
+              title="Expenses"
+              markerType="Hidden"
+            />
+
+            <IgrSplineAreaSeries
+              name="series3"
+              xAxisName="xAxis"
+              yAxisName="yAxis2"
+              valueMemberPath="IncomePerRevenue"
+              title="Income / Revenue %"
+              markerType="Circle"
+            />
+
+            <IgrCalloutLayer
+              name="callout1"
+              targetSeriesName="series1"
+              xMemberPath="RevenueX"
+              yMemberPath="Revenue"
+              labelMemberPath="FormattedRevenue"
+              isAutoCalloutBehaviorEnabled={false}
+              useValueForAutoCalloutLabels={false}
+              calloutLeaderBrush="Transparent"
+              calloutBackground="Transparent"
+              calloutTextColor="Black"
+              calloutPositionPadding="-5"
+            />
+
+            <IgrCalloutLayer
+              name="callout2"
+              targetSeriesName="series2"
+              xMemberPath="ExpensesX"
+              yMemberPath="Expenses"
+              labelMemberPath="FormattedExpenses"
+              isAutoCalloutBehaviorEnabled={false}
+              useValueForAutoCalloutLabels={false}
+              calloutLeaderBrush="Transparent"
+              calloutBackground="Transparent"
+              calloutTextColor="Black"
+              calloutPositionPadding="-5"
+            />
+
+            <IgrCalloutLayer
+              name="callout3"
+              targetSeriesName="series3"
+              isAutoCalloutBehaviorEnabled={true}
+              useValueForAutoCalloutLabels={true}
+              calloutLeaderBrush="Transparent"
+              calloutTextColor="Black"
+              calloutBackground="LightGray"
+              autoCalloutLabelPrecision={1}
```

Comment:
can you place property settings of ALL series/layer on 2-3 lines instead of on each line? This way, sample code looks like is smaller and easier to understand

Reply:
done. 
https://github.com/IgniteUI/igniteui-react-examples/pull/125/commits/cd7e33ff17c101db72f69c2554b757de05855b87